install.sh

```diff
@@ -7,10 +7,10 @@ if [ -z "$CLIENT" ] && [ -z "$SERVER" ]; then
     CLIENT=y
 fi
 
-if [ "$USER" = "root" ]; then
+if [ "`whoami`" = "root" ]; then
```

Comment:
Again, not sure if this is intentional, but it seems to remove the ability to specify $USER as arg. [Some scripts](https://github.com/LetheanMovement/lethean-vpn/blob/master/server/easy-deploy-node.sh) use $USER